File: move_files.js
const fs = require('fs');
const path = require('path');
const base_dir = 'd:\\wrokspace\\takamasu';
const target_dir = path.join(base_dir, 'takamasu');

if (!fs.existsSync(target_dir)) {
    fs.mkdirSync(target_dir);
}

fs.readdirSync(base_dir).forEach(file => {
    if (['takamasu', 'README.md', '.git', 'move_files.js', 'move_files.py'].includes(file)) {
        return;
    }
    
    // Fix mangled file name
    let targetName = file;
    if (file.includes('1怨듯빆') || file.includes('1공항')) {
        targetName = '1공항 atm.jpg';
    }
    
    const src = path.join(base_dir, file);
    const dst = path.join(target_dir, targetName);
    
    try {
        fs.renameSync(src, dst);
        console.log(`Moved ${file} to ${dst}`);
    } catch (e) {
        console.error(`Failed to move ${file}: ${e}`);
    }
});

File: move_files.py
import os
import shutil

base_dir = r"d:\wrokspace\takamasu"
target_dir = os.path.join(base_dir, "takamasu")
if not os.path.exists(target_dir):
    os.makedirs(target_dir)

for file in os.listdir(base_dir):
    if file in ["takamasu", "README.md", ".git", "move_files.py"]:
        continue
    src = os.path.join(base_dir, file)
    dst = os.path.join(target_dir, file)
    try:
        shutil.move(src, dst)
        print(f"Moved {file}")
    except Exception as e:
        print(f"Failed to move {file}: {e}")
